### DISCUSSION TITLE
Welcome to caseStudy1 Discussions!

### DISCUSSION THREAD

**Discussion by gindevp** [quyet0709/caseStudy1#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by quyet79** [comment]
hello me, me and you, let's discuss :D

**Reply by gindevp** [reply]
ok bro :D

**Comment by quyet79** [comment, marked as answer]
why don't we improve this website more :D

**Reply by gindevp** [reply]
I'll probably do it later

**Comment by gindevp** [comment]
we don't have time brother, maybe later we will improve more :D
